packages/cxl-lumo-styles/package.json

```diff
@@ -13,7 +13,6 @@
     "@polymer/iron-iconset-svg": "^3.0.1",
     "@vaadin/vaadin-lumo-styles": "^1.5.0",
     "@vaadin/vaadin-themable-mixin": "^1.5.2",
-    "lit-element": "^2.2.1",
-    "lit-html": "^2.0.0-rc.3"
+    "lit-element": "^2.2.1"
```

Comment:
Why no lit upgrade here?

Reply:
There's a fairly outdated `instanceof` check that relies on Vaadin's own implementation (https://github.com/vaadin/vaadin-themable-mixin/blob/v1.5.2/register-styles.html#L80), that when results are passed through, fails. I feel the smoothest upgrade path is to keep it as the bundle size isn't impacted that much.

Reply:
> There's a fairly outdated `instanceof` check :
> 
> https://github.com/conversionxl/aybolit/blob/41c200a55833dfdec98daa937ef3d73dc8854581/packages/cxl-lumo-styles/src/utils.js#L17

I think in current PR, we can keep it, can phase out later.

Reply:
@anoblet 

Found this: https://github.com/vaadin/docs/blob/ae5785e65c01d10384a203f711686d0fa780eaac/frontend/demo/foundation/include-module.ts#L1
https://github.com/vaadin/web-components/blob/1f9603fcba7d56f16b49e211339f802cd51ae7c0/packages/vaadin-themable-mixin/vaadin-themable-mixin.d.ts#L7
https://github.com/vaadin/web-components/pull/360/files
https://github.com/vaadin/web-components/blob/0abca1c5db71473149c8833be442e979385433a5/packages/vaadin-themable-mixin/vaadin-themable-mixin.js#L6

Wouldn't it work in our case?

Reply:
When the SASS gets built, it gets wrapped in the `css` tag.

That process I believe happens here: https://github.com/conversionxl/aybolit/blob/anoblet/feat/lit/packages/sass-render/sass-template.tmpl

I've tried adding the newest `@vaadin/vaadin-themable-mixin` to the `sass-render` package with no luck.

I also tried swapping this line:

```
import { css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
```
with
```
import { css } from 'lit';
```

Which also did not work.

I'm not sure how to get the signature of the wrapping css tag to match the css tag from lit.

Reply:
> When the SASS gets built, it gets wrapped in the `css` tag.
> 
> That process I believe happens here: [https://github.com/conversionxl/aybolit/blob/anoblet/feat/lit/packages/sass-render/sass-template.tmpl](https://github.com/conversionxl/aybolit/blob/anoblet/feat/lit/packages/sass-render/sass-template.tmpl?rgh-link-date=2022-06-13T11%3A07%3A53Z)
> 
> I've tried adding the newest `@vaadin/vaadin-themable-mixin` to the `sass-render` package with no luck.
> 
> I also tried swapping this line:
> 
> ```
> import { css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
> ```
> 
> with
> 
> ```
> import { css } from 'lit';
> ```
> 
> Which also did not work.
> 
> I'm not sure how to get the signature of the wrapping css tag to match the css tag from lit.

Ok thanks. I will look into this.

Reply:
I'm fine with getting rid of this check.

Reply:
Revert this commit: https://github.com/conversionxl/aybolit/pull/211/commits/152d778425f5307ab63828f57d846111fb28af8f

I have checked the code, and it can't be resolved in this PR. In vaadin upgrade PR we can upgrade this.

Reply:
@saas786 Done